packages/components/src/color-picker/inputs.js

```diff
@@ -178,7 +178,7 @@ export class Inputs extends Component {
 			return (
 				<fieldset>
 					<VisuallyHidden as="legend">
-						{ __( 'Color value in RGB' ) }
+						{ __( 'Color value in RGBA' ) }
```

Comment:
Do you think this should stay "RGB" unless `disableAlpha` is false?

Reply:
You're right. I missed that. Thanks!